### ISSUE TITLE
Avoid littering git repository when running locally

### ISSUE THREAD

**Issue by avillar** [opengeospatial/ogc-na-tools#3]
Files generated/modified by `update_vocabs.py` are tracked by git, which is necessary when running the pipelines but gets confusing when working locally.

**Comment by avillar** [comment]
@rob-metalinkage, I have added an output directory argument to the script. We can now run it locally like this:

```
python -m ogc.na update_vocabs.py -o build <rest of args>
```

and all new files (validation outputs, entailed files) will be put under that `build` subdirectory, recreating the path structure from the working directory (e.g., `definitions/x/y.ttl` is entailed into `build/definitions/x/entailed/y.ttl`). We could also use an external directory (like `/tmp/build`). 

When running from the pipe, we just leave the output directory option out, and it works like it always has.

**Comment by rob-metalinkage** [comment]
Good idea.. we can revisit if we need to test redirection to these locally... but that is a bigger issue to consider.

**Comment by avillar** [comment]
We could also create a simple cleanup.py script that finds all entailed and 
validation results files and undoes the changes (using git checkout / git 
clean / rm)...
--
Enviado desde mi teléfono.
